integration_tests/fixtures/liteCategoriser/supervisorView.ts

```diff
@@ -15,7 +15,7 @@ export const supervisorViewSeedData: LiteCategoryDbRow[] = [
     approved_by: null,
     assessment_committee: 'RECP',
     assessment_comment: 'comment',
-    next_review_date: moment().add(6, 'months').startOf('day').toISOString(),
+    next_review_date: moment().add(6, 'months').toISOString(),
```

Comment:
odd one, this is what flagged the test as failing, but when fixed the other issue started below